django/applications/catmaid/control/sampler.py

```diff
@@ -965,14 +965,14 @@ def add_multiple_sampler_domains(request, project_id, sampler_id):
             "ends": [{
                 "id": e.id,
                 "node_id": e.end_node_id
-            } for e in domain_ends] # FIXME: domain_ends is not defined in this function
```

Comment:
we fixed this but I forgot to remove this FIXME in the last diff

Reply:
Indeed.